src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -490,5 +494,58 @@ protected void SetLicenseLimitResponse(LicenseLimit licenseLimit)
                 writer.Flush();
             }
         }
+
+        protected async Task DatabaseConfigurations(Func<TransactionOperationContext, string,
```

Comment:
This does not belong in the base class, move it to the dedicated handler

Reply:
We can move it to DatabaseRequestHandler